app/views/templates/_template.html.slim

```diff
@@ -1,6 +1,6 @@
 tr.info
   td
-    =link_to template_path(template)
+    =link_to template_path(template), rel: 'tooltip', title: 'Show template'
       = template.name
   td
     .btn-toolbar aria-label="..." role="toolbar"
```

Comment:
pls make the same buttons like in _question.html.slim and give them tooltips too.